pkgs/development/libraries/bullet/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, fetchFromGitHub, cmake, mesa, freeglut }:
+{ stdenv, fetchFromGitHub, cmake, mesa, freeglut, darwin }:
```

Comment:
I prefer to inherit the frameworks into the callPackage scope instead of using the `darwin` attribute directly.